addOns/webdrivers/webdriverlinux/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 - Fix link in help page.
 
+## 8 - 2019-04-5
```

Comment:
This should be removed, the changes should be added to the Unreleased section.

Reply:
@thc202 thank you, fixed it.